test/spec/bidmanager_spec.js

```diff
@@ -264,7 +264,7 @@ var bidmanager = require('../../src/bidmanager');
                         }
                 };
 
-                var expected = {"hb_bidder":  bidderCode, "hb_adid": adId,"hb_pb": 15.0 };
```

Comment:
Great!  I assumed it was just a quick one so sorry for being pedantic before you've even had time to clean it up.  Thanks